### PR TITLE
fix: lg editor validation error & intelligence

### LINKED ISSUE #3815: Inline lg validation is wrong on import file
## Browser

- [ ] Electron distribution
- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [x] macOS
- [ ] Windows
- [ ] Ubuntu

## Screenshots
image

### PR DESCRIPTION
**Pull request by zhixzhan**
## Description

fix lg editor validation error & intelligence, bug introduced by #3738 & #3337 

## Task Item
close #3815

## Screenshots
image

### CODE REVIEW

**Comment by boydc2014** [comment]
Can you add some details about the cause and the fix to better understand the context

**Comment by boydc2014** [comment]
I can repro the issue without this fix and confirm this PR fixed it.

**Comment by zhixzhan** [comment]
intelligence not work is because this line only return templates of current lg file, but we need is allTemplate(include imported file)
https://github.com/microsoft/BotFramework-Composer/pull/3337/files#r467668623

File: Composer/packages/tools/language-servers/language-generation/src/LGServer.ts [hunk 1]
```diff
@@ -166,7 +166,8 @@ export class LGServer {
         const lgTextFile = lgTextFiles.find((item) => item.id === fileId);
         if (lgTextFile) {
           const lgFile = lgUtil.parse(lgTextFile.id, lgTextFile.content, lgTextFiles);
-          return lgUtil.updateTemplate(lgFile, templateId, { body: content });
```
Comment (zhixzhan) [inline Composer/packages/tools/language-servers/language-generation/src/LGServer.ts:169]: lgUtil.updateTemplate now accept `resolver` and return `lgFile`, here miss `resolver` then diagnostic show imported file not find

**Pull request merged**